plutus-conformance/test-cases/uplc/evaluation/builtin/semantics/writeBits/case-32/case-32.uplc

```diff
@@ -7,6 +7,6 @@
    ]
    (con (list integer) [340, 342, 343])
   ]
-  (con (list bool) [True, True, True, False, False, False]) -- Only the leftmost values are used if the lists are of different lengths.
+  (con bool False)
```

Comment:
This should `con bool True`, not `con bool False`, if we want to preserve the old behavior.

Reply:
That would make it a duplicate of [case-30.uplc](https://github.com/IntersectMBO/plutus/pull/6547/files#diff-7a8d00cb3164bd4ea936f09ca30b732480c73e65ac09aef539878e5522a4b1cb).  This test was testing something that doesn't apply any more and instead of deleting it and renaming the others I was lazy and changed it to test something else that I don't think we were testing elsewhere (or at least not exactly).

Reply:
Fair enough.